spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/slf4j/Slf4jSpanStartedListener.java

```diff
@@ -0,0 +1,25 @@
+package org.springframework.cloud.sleuth.slf4j;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.slf4j.MDC;
+import org.springframework.cloud.sleuth.Span;
+import org.springframework.cloud.sleuth.Trace;
+import org.springframework.cloud.sleuth.event.SpanStartedEvent;
+import org.springframework.context.ApplicationListener;
+
+/**
+ * @author Spencer Gibb
+ */
+@Slf4j
+public class Slf4jSpanStartedListener implements ApplicationListener<SpanStartedEvent> {
+
+	@Override
+	public void onApplicationEvent(SpanStartedEvent event) {
+		Span span = event.getSpan();
+		//TODO: what log level?
+		log.info("Starting span: {}", span);
```

Comment:
In our production cases INFO was really a good thing cause we saw where the initalization of the whole business process took place. 

On the other hand we had to introduce filtering if the URIs cause Swagger users flooded our logs with meaningless entries. But after setting the patterns it's really ok right now. So IMO it should be info